static/src/js/components/Errors/ErrorBoundary.js

```diff
@@ -53,9 +53,9 @@ class ErrorBoundary extends Component {
     if (hasError) {
       return (
         <div className="wrap">
-          <h1>
+          <h2 className="h1">
```

Comment:
just curious why h2 with a className of h1.  why not just use h1?

Reply:
Its an SEO thing. Its best to have a single h1 on the page. The h1 for our application will always be the site logo. We had a few spots where we were using additional h1 tags. In order to prevent any visual regression, I am just using the bootstrap `h1` to mimic what was in place before.